Authenticator/Source/AppController.swift

```diff
@@ -43,8 +43,15 @@ class AppController {
     }()
 
     init() {
+        if Process.isDemo {
+            store = DemoTokenStore()
```

Comment:
I had to figure out all the different broken pieces of the Xcode 8 simulator keychain for https://github.com/mattrubin/OneTimePassword/issues/76. It was a mess, but it mostly works now.

I do like the idea of a working in-memory token store. That will be useful when I get around to adding better tests.